helmfile/values/opensearch/slm.yaml.gotmpl

```diff
@@ -0,0 +1,26 @@
+{{ if not (or (eq .Values.objectStorage.type "s3") (eq .Values.objectStorage.type "gcs") ) }}
```

Comment:
I don't see us using these variables in here, do we still need the check for this file?

Reply:
It is carried over from elasticsearch-slm, I guess the check makes sense since it requires that OpenSearch is set up with a snapshot repository, which we only do with object storage.